shared/js/background/declarative-net-request.js

```diff
@@ -217,7 +220,7 @@ async function updateExtensionConfigRules (etag = null, configValue = null) {
 
     if (!configValue) {
         await tdsStorage.ready('config')
-        configValue = await tdsStorage.getListFromLocalDB('config')
+        configValue = await tdsStorage.config
```

Comment:
@kzar This line was causing issues in the tests, where we're loading a config and then updating the allowlist in quick succession. In this case we'd get invalid data from the DB, because the data didn't get flushed yet, causing the config rule generation to throw. I hope it's OK to just load the live version of the config here, or was there another reason to go to the DB here?